packages/client-core/src/admin/components/Project/UpdateDrawer.tsx

```diff
@@ -166,6 +177,16 @@ const UpdateDrawer = ({ open, builderTags, onClose }: Props) => {
           {t('admin:components.project.updateEngine')}
         </DialogTitle>
 
+        <div className={styles.helmSubheader}>
+          <div>{t('admin:components.setting.helm.mainHelmToDeploy')}</div>:
+          <a href="/admin/settings#helm">{helmSetting?.main.length > 0 ? helmSetting.main : 'Current Version'}</a>
+        </div>
+
+        <div className={styles.helmSubheader}>
+          <div>{t('admin:components.setting.helm.builderHelmToDeploy')}</div>:
+          <a href="/admin/settings#helm">{helmSetting?.builder.length > 0 ? helmSetting.builder : 'Current Version'}</a>
```

Comment:
Current Version. The behavior if there is no version saved in the new settings is to get and use the currently-deployed version of the Helm chart in question, so that deployments don't get silently updated to a newer version without their knowledge that might break the deployment process.